package.nls.json

```diff
@@ -194,6 +194,10 @@
     },
     "cmake-tools.configuration.cmake.launchBehavior.description": "Controls what happens with the launch terminal when you launch a target.",
     "cmake-tools.configuration.cmake.automaticReconfigure.description": "Automatically configure CMake project directories when the kit or the configuration preset is changed.",
+    "cmake-tools.debugger.pipeName.description": "The named piped that the debugger will communicate with CMake with.",
```

Comment:
Awkward grammar. What about "Name of the pipe (on Windows) or domain socket (on Unix) to use for debugger communication."? (taken from the CMake docs)

Reply:
Changed. 👍